src/components/Restaurant.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div>
+    <div class="restaurant" v-if="restaurants.length > 0">
+      <div
+        class="restaurant__img"
+        v-if="restaurant.restaurant.featured_image.length > 0"
```

Comment:
Can `featured_image` be `undefined`? What then? Are you sure you don't just want to compare it to an empty string (`""`)?